src/shadowbox/server/server_config.ts

```diff
@@ -34,6 +35,8 @@ export interface ServerConfigJson {
   // We don't serialize the shadowbox version, this is obtained dynamically from node.
   // Public proxy hostname.
   hostname?: string;
+  // Data transfer limit applied to all access keys.
+  accessKeyDataLimit?: DataLimit;
```

Comment:
We should add a note to the type definition that `DataLimit` should be a JSON object.

Reply:
Done.